test: add basic mobile baseline tests

christian-byrne opened the pull request:
## Summary

Allows authors to visualize how their changes affect mobile view. Often we add some fundamental change to the UI and forget to make it responsive, this test helps keep track of that.

┆Issue is synchronized with this [Notion page](https://www.notion.so/PR-7415-test-add-basic-mobile-baseline-tests-2c76d73d3650810bb4bad5a1f6c7c53c) by [Unito](https://www.unito.io)

christian-byrne committed:
add basic mobile baseline tests

christian-byrne committed:
use the standard approach for clear workflow

christian-byrne committed:
Add next frame

DrJKL commented on browser_tests/tests/mobileBaseline.spec.ts:
```diff
@@ -0,0 +1,23 @@
+import { comfyPageFixture as test } from '../fixtures/ComfyPage'
+import { expect } from '@playwright/test'
+
+test.describe('Mobile Baseline Snapshots', () => {
```
Slight preference for just tagging the suite:
```suggestion
test.describe('Mobile Baseline Snapshots', {
  tag: '@mobile',
}, () => {
```

DrJKL commented on browser_tests/tests/mobileBaseline.spec.ts:
```diff
@@ -0,0 +1,23 @@
+    await comfyPage.nextFrame()
+    await expect(comfyPage.page).toHaveScreenshot('mobile-settings-dialog.png')
+  })
+})
```
Check the Playwright config.

christian-byrne committed:
fix: improve mobile baseline test stability

- Use suite-level @mobile tag via describe.configure
- Add retryable assertion for canvas clear
- Screenshot dialog root instead of full page

christian-byrne committed:
fix: use @mobile tag in test names, add ConfirmClear setting

Commit:
[automated] Update test expectations
